sci-visualization/mantid/files/mantid-3.3.0-minigzip-OF.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/MantidPlot/src/zlib123/minigzip.c b/MantidPlot/src/zlib123/minigzip.c
+index 97fbae1..5f4612f 100644
+--- a/MantidPlot/src/zlib123/minigzip.c
++++ b/MantidPlot/src/zlib123/minigzip.c
+@@ -50,6 +50,10 @@
+ #  include <unix.h> /* for fileno */
+ #endif
+ 
++#ifndef OF
++#  define OF(args) args
++#endif
++
+ #ifndef WIN32 /* unlink already in stdio.h for WIN32 */
+   extern int unlink OF((const char *));
+ #else
```

Comment:
@junghans suggested to clean patches as explained [here](http://dev.gentoo.org/~vapier/clean-patches). I think this would be a good idea.